src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -246,6 +256,7 @@ private void InitializeFromCodeFile(ReviewCodeFileModel file, CodeFile codeFile)
         {
             file.Language = codeFile.Language;
             file.VersionString = codeFile.VersionString;
+            file.Name = codeFile.Name;
```

Comment:
Still need this?

Reply:
Yes, because I removed UpdateRevisionNames